ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,69 @@
+<!--
+************ STOP!! ************
+If this issue identifies a security vulnerability, DO NOT submit it! Instead, contact
+the Apache Software Foundation Security Team at security@trafficcontrol.apache.org and follow the
+guidelines at https://www.apache.org/security/ regarding vulnerability disclosure.
+-->
+
+<!--
+- For *SUPPORT QUESTIONS*, use the
+[Traffic Control slack channels](https://traffic-control-cdn.slack.com) or [Traffic Control mailing lists](http://trafficcontrol.apache.org/mailing_lists/).
+- Before submitting, please **SEARCH GITHUB** for a similar issue or PR. -->
+
+**I'm submitting a ...**
+<!-- (check one with "x") -->
+<!--- security vulnerability (STOP!! - see above)-->
+- [ ] bug report
+- [ ] regression from TC 3.x
+- [ ] new feature / enhancement request
+- [ ] improvement (functionality exists but it could be improved in some way)
```

Comment:
Would examples of "improvement" include refactoring, performance fixes, etc? I.e. doesn't really provide new functionality but isn't necessarily a bug fix? If that's the case, maybe we should say "improvement (e.g. refactoring, performance enhancement, etc -- no new functionality)".

Reply:
what about a tech debt item? and how about changing improvement to:

improvement (functionality exists but it could be improved in some way. i.e. usability, performance, etc.)

Reply:
The current wording sounds good to me